src/Common/tests/System/Collections/TestingTypes.cs

```diff
@@ -147,6 +147,9 @@ public int GetHashCode(int x)
 
     #region TestClasses
 
+#if netstandard17
```

Comment:
Hopefully we don't need it for the source, once you can clean-up some of the build configurations but if you do still have some source then I would be fine with it being a shared stub there as well. Mainly I hate seeing all these #ifdefs.